src/main/javascript/base/SearchRequest.js

```diff
@@ -39,7 +39,16 @@ window.cat.SearchRequest = function (searchUrlParams) {
     }
 
     var _encodeSort = function () {
-        return (!!_sort.property ? 'sort=' + _sort.property + ':' + ((_sort.isDesc === true || _sort.isDesc === 'true') ? 'desc' : 'asc') : '');
+        if(!!_sort.property) {
+            var properties = _sort.property.split(';');
+            var result = 'sort=';
+            for(var i = 0; i< properties.length; i++) {
+                result += properties[i] + ':' + ((_sort.isDesc === true || _sort.isDesc === 'true') ? 'desc' : 'asc') + ';';
+            }
+            return result.slice(0, -1);
```

Comment:
I am actually not sure, I would have to test it. Previously there was no `;` before so i did not want to add one now. I will test this as soon as I have the time.